dockstore-webservice/src/test/java/io/dockstore/webservice/permissions/sam/SamPermissionsImplTest.java

```diff
@@ -214,19 +220,21 @@ public void testSetPermission() throws ApiException {
         Assert.assertEquals(permissions.size(), 1);
     }
 
-    @Test(expected = CustomWebApplicationException.class)
+    @Test
     public void setPermissionTest1() {
         try {
-            when(resourcesApiMock.listResourcePolicies(SamConstants.RESOURCE_TYPE, SamConstants.WORKFLOW_PREFIX + FOO_WORKFLOW_NAME)).thenThrow(new ApiException(500, "Server error"));
+            when(resourcesApiMock.listResourcePolicies(SamConstants.RESOURCE_TYPE, SamConstants.WORKFLOW_PREFIX + FOO_WORKFLOW_NAME)).thenThrow(new ApiException(HttpStatus.SC_INTERNAL_SERVER_ERROR, "Server error"));
             final Permission permission = new Permission();
             permission.setEmail("jdoe@example.com");
             permission.setRole(Role.WRITER);
+            thrown.expect(CustomWebApplicationException.class);
             samPermissionsImpl.setPermission(fooWorkflow, userMock, permission);
             Assert.fail("setPermissions did not throw Exception");
```

Comment:
With thrown.expect, and even with the way I had it before, is this line even needed any more?

Reply:
Hmmm, it was/is probably redundant.
So is the whole try ... catch as well since even before it was catching a different exception.